web/controllers/respondent_controller.ex

```diff
@@ -62,13 +102,9 @@ defmodule Ask.RespondentController do
 
   defp sort_respondents(query, sort_by, sort_asc) do
     case {sort_by, sort_asc} do
-      {"phoneNumber", "true"} ->
-        query |> order_by([r], asc: r.hashed_number)
-      {"phoneNumber", "false"} ->
-        query |> order_by([r], desc: r.hashed_number)
-      {"date", "true"} ->
+      {"updated_at", "true"} ->
```

Comment:
No, it isn't. But I'll revert this change. Because it adds more confusion to other related issues we already have.

[Here](https://github.com/instedd/surveda/blob/master/web/views/respondent_view.ex#L17-L20) we're rendering the last response date of every respondent.

And [here](https://github.com/instedd/surveda/blob/master/web/controllers/respondent_controller.ex#L69-L72) we're sorting by the last respondent update.

We're rendering and sorting by different things. It's a bug, isn't it?

Reply:
Ok, revert this change, and create a separate tech-debt issue to unify what data expose as respondent date.